ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1182,25 +1174,19 @@ tests:
   steps:
     cluster_profile: aws-qe
     env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
```

Comment:
@JoelSpeed same above comment. just need to add `FEATURE_SET: TechPreviewNoUpgrade`, no need to delete existing env.

cc @jianzhangbjz